gtk4/tests/builder_rust_scope.rs

```diff
@@ -0,0 +1,76 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use gtk4 as gtk;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+
+#[test]
+fn builder_rust_scope() {
+    gtk::init().unwrap();
```

Comment:
that won't work, see the other tests in gtk/src/bitset_iter for a way to do it that would work properly across various tests (you can init gtk only once).

Reply:
Good call, I'm using that and I fixed it up to work with `#[should_panic]` as well.